content/post/52-go-ipfs-0.4.18.md

```diff
@@ -0,0 +1,283 @@
+---
+date: 2018-11-01
+url: 52-go-ipfs-0.4.18
+title: go-ipfs 0.4.18 released
+author: Steven Allen and Erik Ingenito
+---
+
+[go-ipfs 0.4.18](https://dist.ipfs.io/#go-ipfs) has been released! This is one of largest go-ipfs releases to date; 3 months in the making. *Thanks to all our contributors for your awesome work!*
+
+## Highlights
+The headline features this release are experimental [QUIC support](#quic), the [gossipsub](#pubsub)
+pubsub routing algorithm, [pubsub message signing](#pubsub), a refactored `ipfs p2p`
+[command](#commands-changes), and a new [WebUI](#webui). However, that's just scratching the surface.
+
+
+## QUIC
+First up, on the networking front, this release has also introduced experimental
+support for the QUIC protocol. QUIC is a new UDP-based network transport that
+solves many of the long standing issues with TCP.
+
+For us, this means (eventually):
+
+* **Fewer local resources**. TCP requires a file-descriptor per connection while
+  QUIC (and most UDP based transports) can share a single file descriptor
+  between all connections. This should allow us to dial faster and keep more
+  connections open.
+* **Faster connection establishment**. When client authentication is included, QUIC
+  has a three-way handshake like TCP. However, unlike TCP, this handshake brings
+  us all the way from 0 to a fully encrypted, authenticated, and
+  multiplexed connection. In theory (not yet in practice), this should
+  significantly reduce the latency of DHT queries.
+* **Behaves better on lossy networks**. When multiplexing multiple requests over a
+  single TCP connection, a single dropped packet will bring the entire
+  connection to a halt while the packet is re-transmitted. However, because QUIC
+  handles multiplexing internally, dropping a single packets affects only the
+  related stream.
+* **Better NAT traversal**: NAT hole-punching is significantly easier and, in
+  many cases, more reliable with UDP than with TCP.
+
+However, we still have a long way to go. While we encourage users to test this,
+the IETF QUIC protocol is still being actively developed and *will* change. You can find instructions for enabling it [here](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#QUIC).
+
+## Pubsub
+
+go-ipfs now supports the [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) routing algorithm and message signing.
+
+The gossipsub routing algorithm is *significantly* more efficient than the
+current floodsub routing algorithm. Even better, it's fully backwards compatible
+so you can enable it and still talk to nodes using the floodsub algorithm. You
+can find instructions to enable gossipsub in go-ipfs
+[here](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#gossipsub).
+
+Messages are now, finally, **signed by their authors**. While signing has been
```

Comment:
Let's use "is now" instead of "has been" so as to be clear about when that support was added (aka in this release!)

> "While signing **is now** enabled by default..."